spring-data-jdbc/src/test/java/org/springframework/data/jdbc/repository/JdbcRepositoryCrossAggregateHsqlIntegrationTests.java

```diff
@@ -56,7 +58,8 @@
 
 	@Configuration
 	@Import(TestConfiguration.class)
-	@EnableJdbcRepositories(considerNestedRepositories = true)
+	@EnableJdbcRepositories(includeFilters = @Filter(type = FilterType.REGEX,
```

Comment:
We should improve handling for non-existent entity types in `JdbcRepositoryFactory.getTargetRepository(…)` (e.g. calling `getRequiredPersistentEntity(…)` instead of `getPersistentEntity(…)`).

```
java.lang.IllegalArgumentException: entity is marked non-null but is null
	at org.springframework.data.jdbc.repository.support.SimpleJdbcRepository.<init>(SimpleJdbcRepository.java:36)
```

is hard to debug.

Reply:
✓